final_prep/mod_zero_hero.rb

```diff
@@ -2,51 +2,92 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = "Iceman"
+special_ability = "Ice control"
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "Hello #{hero_name}."
+catchphrase = "Please don't use your ${special_ability} on me!"
+
 # Declare two variables - power AND energy - set to integers
 
+power = 1000
+energy = 500
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy + 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
+arch_enemies = ["Dr. Evil", "Horse-head Jones", "The Demonizer"]
+sidekicks = ["Mr.Splendor", "The Green Hammer", "Angela Simmons"]
+
 # Print the first sidekick to your terminal
+puts sidekicks[0]
 
 # Print the last arch_enemy to the terminal
 
+puts arch_enemies[2]
+
 # Write some code to add a new arch_enemy to the arch_enemies array
 
+arch_enemies << "The Priest"
+
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
 
+puts arch_enemies
+
 # Remove the first sidekick from the sidekicks array
 
+sidekicks.delete_at(0)
+
+
 # Print the sidekicks array to terminal to ensure you added a new sidekick
 
+puts sidekicks
+
 # Create a method called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
 
 # Your method should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
 
+def assess_situation(danger_level, save_the_day, bad_excuse)
```

Comment:
Beautiful style and syntax, your code is very readable.

🤔 is there a way you can modify your solution to not require the `and` or any direct replacement for it?